manifests/database.pp

```diff
@@ -0,0 +1,15 @@
+# sets up separate postgresql instance for pulpcore
+# @api private
+class pulpcore::database {
+
+  class { 'postgresql::globals':
```

Comment:
Generally this is not a good idea. This will break others if they set the globals. In the installer we set this via Hiera.